lib/excoveralls/cobertura.ex

```diff
@@ -188,7 +188,7 @@ defmodule ExCoveralls.Cobertura do
     # We use Range.new/3 because using x..y//step would give a syntax error on Elixir < 1.12
     defp get_slice_range_for_package_name(c_path), do: Range.new(String.length(c_path) + 1, -1, 1)
   else
-    defp get_slice_range_for_package_name(c_path), do: (String.length(c_path) + 1)..-1
+    defp get_slice_range_for_package_name(c_path), do: (String.length(c_path) + 1)..-1//1
```

Comment:
Well, why not use the `Range.new` syntax directly then?
```diff
-  # TODO: Remove when we require Elixir 1.12 as minimum and inline it with range syntax
-  if Version.match?(System.version(), ">= 1.12.0") do
-    # We use Range.new/3 because using x..y//step would give a syntax error on Elixir < 1.12
-    defp get_slice_range_for_package_name(c_path), do: Range.new(String.length(c_path) + 1, -1, 1)
-  else
-    defp get_slice_range_for_package_name(c_path), do: (String.length(c_path) + 1)..-1//1
-  end
+  # TODO: Update when we require Elixir 1.12 as minimum to use the inline range syntax `x..y//z`
+  defp get_slice_range_for_package_name(c_path), do: Range.new(String.length(c_path) + 1, -1, 1)
```

And sorry for not catching the error when I created the PR. As the actions didn't run, I didn't expect it to fail as it was behind a ward already :D